docs/tutorial/work-pools.md

```diff
@@ -0,0 +1,306 @@
+│         - Create an Artifact Registry repository named prefect-images                                    │
+│         - Create a service account for managing Cloud Run jobs: prefect-cloud-run                        │
+│             - Service account will be granted the following roles:                                       │
+│                 - Service Account User                                                                   │
+│                 - Cloud Run Developer                                                                    │
+│         - Create a key for service account prefect-cloud-run                                             │
+│                                                                                                          │
+│     Updates in Prefect workspace                                                                         │
+│                                                                                                          │
+│         - Create GCP credentials block my--pool-push-pool-credentials to store the service account key   │
+│                                                                                                          │
+╰──────────────────────────────────────────────────────────────────────────────────────────────────────────╯
+Proceed with infrastructure provisioning? [y/n]: y
+Activating Cloud Run API
+Activating Artifact Registry API
+Creating Artifact Registry repository
+Configuring authentication to Artifact Registry
+Setting default Docker build namespace
+Creating service account
+Assigning roles to service account
+Creating service account key
+Creating GCP credentials block
+Provisioning Infrastructure ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 100% 0:00:00
+Infrastructure successfully provisioned!
+Created work pool 'my-cloud-run-pool'!
+```
+
+</div>
+
+After infrastructure provisioning completes, you will be logged into your new Artifact Registry repository and the default Docker build namespace will be set to the URL of the repository.
+
+While the default namespace is set, any images you build without specifying a registry or username/organization will be pushed to the repository.
+
+To take advantage of this functionality, you can write your deploy scripts like this:
+
+```python hl_lines="14" title="example_deploy_script.py"
+from prefect import flow                                                       
+from prefect.deployments import DeploymentImage                                
+
+
+@flow(log_prints=True)
+def my_flow(name: str = "world"):
+    print(f"Hello {name}! I'm a flow running on Cloud Run!")
+
+
+if __name__ == "__main__":                                                     
+    my_flow.deploy(                                                            
+        name="my-deployment",
+        work_pool_name="above-ground",
+        image=DeploymentImage(
+            name="my-image:latest",
+            platform="linux/amd64",
+        )
```

Comment:
It might be worth including a note about the usages of `DeploymentImage` and `platform`. You only need to include `platform="linux/amd64` if you're building your image on a machine with an ARM-based processor. Otherwise, you could just pass `image="my-image:latest"`.